.github/workflows/publish-swaggerhub.yaml

```diff
@@ -31,17 +31,6 @@ on:
         required: false
         type: string
 
-  workflow_dispatch:
```

Comment:
please revert all changes in this file.

Reply:
Sorry about that mess. I can only guess this got messed between the many rebases. Fixed.